build_product

```diff
@@ -166,6 +168,9 @@ parse_commandline()
 				print_help
 				exit 0
 				;;
+			-p|--profiling)
```

Comment:
We should add your new feature to the documentation. You can reuse / get inspired by the text that you wrote in the pull request description.

Reply:
I added a section called "Profiling the buildsystem" to files 2 and 5 of the documentation, with the former containing a brief overview of how to use the -p|--profiling switch with the build_product script and what it is used for, while the latter contains details on how the two helper scripts build_profiler.sh and build_profiler_report.py actually work.